includes/class-wc-payments.php

```diff
@@ -605,6 +605,9 @@ public static function init() {
 			WC_Payments_Subscriptions::init( self::$api_client, self::$customer_service, self::$order_service, self::$account );
 		}
 
+		// Load the WCPay Subscriptions migration class.
+		include_once WCPAY_ABSPATH . '/includes/subscriptions/class-wc-payments-subscriptions-migrator.php';
```

Comment:
This is temporary to make it easier to test this new class. We'll likely change how this class file is loaded as we further develop the migrator

Reply:
I decided to add a feature flag to this PR: [16ab442](https://github.com/Automattic/woocommerce-payments/pull/6706/commits/16ab4423122b562f31675add9638367abbd6305d)